content/blog/2025/jb-for-communities/index.md

```diff
@@ -11,26 +11,28 @@ featured: false
 draft: false
 ---
 
-A core component of our mission to make research and education more _impactful_, _accessible_, and _delightful_ is leveraging our unique [global network of communities][network] to help us make meaningful improvements to the open-source tools that power their work. In this way, our experience of learning from one community can be used to provide value to our entire network e.g. [our work with PACE on speeding up their CNN model training][pace-gpu].
+A core component of our mission to make research and education more _impactful_, _accessible_, and _delightful_ is leveraging our unique [global network of communities][network] to make meaningful improvements to the open-source tools that power their work. Learning from one community can then provide value to our entire network, e.g., [our work with PACE on speeding up their CNN model training][pace-gpu].
 
-At the heart of the work that our communities perform is the fundamental importance of sharing new findings, best-practice, and community resources. Across our network, we have seen communities creating their own "books" that provide a home for this kind of content. Replicated across many of these books is the concept of a "landing page" that welcomes new members, establishes an identity, and provides jumping off points (or "calls to action") into more detailed resources.
+Central to our communities' work is the importance of sharing new findings, best practices, and community resources. Across our network, we have seen communities creating their own "books" that provide a home for this kind of content. Many of these books feature the concept of a "landing page" that welcomes new members, establishes an identity, and provides jumping-off points (or "calls to action") to more detailed resources.
 
-Until now, each community has been required to understake this work themselves. 2i2c believes that by [building upon existing open-source tools][open-tech] like [Jupyter Book 2][jb-next], we can help communities to focus on the _content_ of their home, without having to spend so much time worrying about how to make that content look _good_. To that effect, we have been working on [an initiative][initiative] to allow communities to more rapidly build interactive starter documentation and provide their users with a rich, interactive and informative onboarding experience. Through this initiative, we have:
+Until now, each community has been required to undertake this work independently. 2i2c believes that by [building upon existing open-source tools][open-tech] like [Jupyter Book 2][jb-next], we can help communities focus on the _content_ of their home, rather than spending time worrying about its _appearance_. To that end, we have been working on [an initiative][initiative] to allow communities to rapidly build interactive starter documentation and provide users with a rich, interactive, and informative onboarding experience. Through this initiative, we have:
```

Comment:
`understake` -- no, that's an intentional new-word I just created. /s ;)

Reply:
hahaha that's the one that triggered this mini update :-D

Reply:
for some reason it reminded me of https://undertale.com/